go.mod

```diff
@@ -3,21 +3,15 @@ module github.com/ory/kratos
 go 1.19
 
 replace (
-	github.com/bradleyjkemp/cupaloy/v2 => github.com/aeneasr/cupaloy/v2 v2.6.1-0.20210924214125-3dfdd01210a3
-
 	github.com/go-sql-driver/mysql => github.com/go-sql-driver/mysql v1.7.0
 
-	github.com/gorilla/sessions => github.com/ory/sessions v1.2.2-0.20220110165800-b09c17334dc2
-	github.com/knadh/koanf => github.com/aeneasr/koanf v0.14.1-0.20211230115640-aa3902b3267a
-
-	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.14.7-0.20210414154423-1157a4212dcb
-	github.com/oleiade/reflections => github.com/oleiade/reflections v1.0.1
```

Comment:
Please change module rewrites only with explicit reasoning and potentially as part of a separate PR. The sessions and koanf forks contain important code changes which fix memory leaks, add missing features, or fix other behavior. Thanks :)

Reply:
Reverted. But they're so annoying to look at 😅